### DISCUSSION TITLE
colab help please

### DISCUSSION THREAD

**Discussion by rafiislambd** [MoonRide303/Fooocus-MRE#95] [General]
how to add another sdxl model?? i can't add another sdxl model..
please fix this option..thanks

**Comment by 3Diva** [comment]
What do you mean you can't add another SDXL model? If you mean another checkpoint other than the base SDXL checkpoint you can add them to your checkpoints folder (Fooocus-MRE\models\checkpoints) and they should show up under checkpoint dropdown menu. You'll need to completely close down and restart Fooocus-MRE in order for it to show up.

**Reply by rafiislambd** [reply]
old code 

-- !aria2c --console-log-level=error -c -x 16 -s 16 -k 1M https://huggingface.co/rafiislam/RakibUniverse/resolve/main/rakibuniversev14.safetensors -d /content/Fooocus/models/checkpoints -o rakibuniversev14.safetensors

but colab update version same code not working i mean not downloaded
